draft-ietf-rats-architecture.md

```diff
@@ -1451,6 +1451,11 @@ Security protections in RATS may be applied at different layers, whether by a co
 This architecture expects conceptual messages to be end-to-end protected based on the role interaction context.
 For example, if an Attester produces Evidence that is relayed through some other entity that doesn't implement the Attester or the intended Verifier roles, then the relaying entity should not expect to have access to the Evidence.
 
+The RATS architecture allows for an entity to function in multiple roles
+({{rolesentities}}) and for composite devices ({{compositedevice}}).
+Implementers need to evaluate their designs to ensure that the assumed security properties of the individual components and roles still holds despite the lack of separation, and that emergent risk is not introduced.
+The specifics of this evaluation will depend on the implementation and use case is out of scope for this document.
+
```

Comment:
Isolation mechanisms in software or hardware that separate Attesting Environments and Target Environments {{twotypes}} can support implementer's evalutaion and design decisions.